.github/workflows/documentation.yml

```diff
@@ -0,0 +1,54 @@
+# This is a workflow to generate the documentation automatically.
+
+name: Generate Documentation
+
+# Controls when the action will run. Triggers the workflow on push
+# events but only for the master branch
+on:
+  push:
+    branches:
+      - master
+      - feature/AutomaticDocDeployment # this will be removed once it's working
+jobs:
+  # This workflow contains a single job called "build"
+  gen_doc:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        java: [11]
+    runs-on: ${{ matrix.os }}
+    name: asciidoctor to html
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Load Cache
```

Comment:
please add a new line so one can see where one steps ends and another one begins.

Reply:
copy